src/content/en/2021/cdn.md

```diff
@@ -63,26 +62,28 @@ These are the limits to our testing [methodology](./methodology):
 
 **CDN detection:** This is primarily done through DNS resolution and HTTP headers. Most CDNs use a DNS CNAME to map a user to an optimal datacenter. However, some CDNs use AnyCast IPs or direct A+AAAA responses from a delegated domain which hide the DNS chain. In other cases, websites use multiple CDNs to balance between vendors, which is hidden from the single-request pass of our crawler.
 
+{# TODO: Should this be in bullet points? I feel like it would be organized a bit better, but open to either way #}
```

Comment:
@Navaneeth-akam

Reply:
I would prefer it to be in paragraph style :)